### DISCUSSION TITLE
how to get this into the v1 editor?

### DISCUSSION THREAD

**Discussion by nicosql** [data-forge/data-forge-ts#171] [Q&A] [answered]
how can i make this an executable desktop app like the old one?

**Comment by ashleydavis** [comment, marked as answer]
Are you talking about Data-Forge Notebook? If so you probably want this repo instead:

https://github.com/data-forge-notebook/data-forge-notebook/

**Reply by nicosql** [reply]
yes! thank you!  halfway through your book :) very helpful in my work, when I get my raise I'll come drop a lump of cash on you lol
